setup.cfg

```diff
@@ -97,3 +108,7 @@ multi_line_output = 0
 balanced_wrapping = True
 include_trailing_comma = false
 length_sort = True
+
+[egg_info]
```

Comment:
fwiw, the existence of this section is incompatible with `setuptools_scm`

Reply:
I don't know where that came from...